src/debugger/channel/AdapterChannel.js

```diff
@@ -105,6 +103,37 @@ export class AdapterChannel {
     this._eventEmitter.addListener(event, listener);
   }
 
+  launch(requestID: number, args: PrepackLaunchArguments, callback: DebuggerResponse => void) {
+    if (!args.prepackCommand || args.prepackCommand.length === 0) {
+      console.error("No command given to start Prepack in adapter");
```

Comment:
Careful of using console.error in testing. We don't want to let error logs escape on tests that succeed.